README.md

```diff
@@ -1,38 +1,65 @@
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
 # About the Project
 ## About The Project
-This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum, be able to view all posts, add a post, and delete a post. Consider adding the ability to update existing posts, add and set categories on blog posts, specify users, etc.
-This app used Redux Toolkit and state slices to manage any state data needed across multiple components. In the case where data is created and used only within one component (or possibly one and a few of its direct children), consider using useState() or useReducer() and traditional prop drilling. Most apps will use a combination of "local" state management (useState() and useReducer()) and "global" state management (Redux or Context API).
-Apply the TypeScript static typing system. try to use TypeScript for the majority of React components! installed typescript, add support for TypeScript linting, and use either .ts or .tsx extension for the files to apply TypeScript type-checking on.
-Use the `BLANK_README.md` to get started.
+<a name="about-the-project"></a>
+This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum, be able to join a game, create a new game, participate with other users in the game, and at the end of the game the scene will display the scores of all participants in the game.
+<p align="left">(<a href="#top">back to top</a>)</p>
 
 ### Built with
+<a name="built-with"></a>
 A few of the frameworks/libraries that were used to build the project are
-* [![React][React.js]][React-url]
+* React
+* Typescript
 * Node.js
 * React Redux 
+* Firebase/Firebase Authentication
 * Axios
 * React Router
 * EsLint and AirBnb
 * Phaser
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+* Socket-io Client
+<p align="left">(<a href="#top">back to top</a>)</p>
 <!-- ### wireframe
   ### Getting Started The App build out with simple wireframes using Figma.
 <img width="328" alt="wire-frame-frontend" src=""> -->
 
 ## Getting Started
-This is an example of how you may give instructions on setting up your project locally.
-To get a local copy up and running follow these simple example steps.
+<a name="getting-started"></a>
+To install the FortNerf App Follow these steps
+
 ### Prerequisites
+<a name="prerequisites"></a>
 * npm
   ```sh
   npm install npm@latest -g
   ```
+
 ### Installation
+<a name="installation"></a>
 _Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
```

Comment:
i really dont think you need this text on #63. seems more like a note or guideline.